core/src/main/java/org/ldaptive/dn/RDn.java

```diff
@@ -0,0 +1,212 @@
+/* See LICENSE for licensing and NOTICE for copyright. */
+package org.ldaptive.dn;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.ldaptive.LdapUtils;
+
+/**
+ * Relative distinguished name containing one or more name value pairs. Name value pairs are ordered from left to right
+ * such that the left-most pair is considered the first. For the RDN 'cn=Jane Doe+mail=jdoe@example.com', the first name
+ * value pair is 'cn=Jane Doe'.
+ *
+ * See <a href="http://www.ietf.org/rfc/rfc4514.txt">RFC 4514</a> for more details on the string representations of
+ * RDNs.
+ *
+ * @author  Middleware Services
+ */
+public class RDn
```

Comment:
The casing of this class looks really odd. Suggest `RDN` given you already have components with a series of capitals, e.g. `DERParser`.